maven/spring-boot-parent/README.md

```diff
@@ -0,0 +1,10 @@
+spring-boot-parent
```

Comment:
please also add an link to this readme in the root README file, probably in an extra section/subsection for spring